LayoutTests/http/tests/storageAccess/request-and-grant-access-cross-origin-sandboxed-iframe-from-prevalent-domain-with-user-interaction-and-access-from-right-frame.https.html

```diff
@@ -124,27 +124,38 @@
                     // Navigate the frame same-site.
                     let existingIframe1 = document.getElementById("TheIframeThatRequestsStorageAccess");
                     existingIframe1.onload = function () {
-                        debug("Did navigate iframe same-site and will now check that it still has storage access.");
+                        debug("Did navigate iframe from a cross-site parent and will now check that it does not have storage access.");
                         let iframe = document.getElementById("TheIframeThatRequestsStorageAccess");
                         iframe.contentWindow.postMessage("reportBackCookies", "https://localhost:8443");
                     };
                     existingIframe1.src = thirdPartyBaseUrl + subPathToGetCookies;
                     break;
                 case "#step8":
                     document.location.hash = "step9";
-                    // Navigate the frame cross-site. This should clear out storage access.
+                    // Re-grant storage access after same-site navigation.
```

Comment:
Do we need another tests for the previous behavior, or do we still have enough coverage for that?

Reply:
We do still test the previous behavior in this test, I just moved it to step9.